### PR TITLE
[I18n] ES locale

### PR DESCRIPTION
**Pull request by xuanxu**
This PR adds all missing translations for the spanish locale

### CODE REVIEW

**Comment by simi** [comment]
:wave: @Lastimoso @acrogenesis @javier-menendez would it be possible to take a look as previous contributors to `es` locale? @deivid-rodriguez feel free to check as well.

File: config/locales/es.yml [hunk 1]
```diff
@@ -197,131 +197,137 @@ es:
...
         sign_out: Salir
-        sign_up: Registrarte
+        sign_up: Registro
       mfa_banner_html:
   mailer:
     confirm_your_email: Por favor confirma tu dirección de correo con el enlace enviado.
-    confirmation_subject: Por favor confirma tu dirección de correo con RubyGems.org
-    link_expiration_explanation_html:
+    confirmation_subject: Por favor confirma tu dirección de correo con %{host}
+    link_expiration_explanation_html: Por favor ten en cuenta que este enlace es válido solo durante 3 horas. Puedes solicitar un enlace actualizado usando la página de <a href='https://rubygems.org/email_confirmations/new'>reenvio del email de confirmación</a>.
```
Comment (javier-menendez) [inline config/locales/es.yml:215]:
```suggestion
    link_expiration_explanation_html: Por favor ten en cuenta que este enlace es válido solo durante 3 horas. Puedes solicitar un enlace actualizado usando la página de <a href='https://rubygems.org/email_confirmations/new'>reenvío del correo de confirmación</a>.
```

File: config/locales/es.yml [hunk 2]
```diff
@@ -197,131 +197,137 @@ es:
...
-      subject:
-      title:
+      subject: Gema %{gem} eliminada de %{host}
+      title: GEMA ELIMINADA
     reset_api_key:
-      subject:
-      title:
-      subtitle:
+      subject: "Clave de API de %{host} restablecida"
+      title: CLAVE DE API RESTABLACIDA
```
Comment (javier-menendez) [inline config/locales/es.yml:254]:
```suggestion
      title: CLAVE DE API RESTABLECIDA
```

File: config/locales/es.yml [hunk 3]
```diff
@@ -197,131 +197,137 @@ es:
...
-      subject:
-      title:
-      subtitle:
+      subject: Aplicación de autenticación desactivada en %{host}
+      title: APLICACIÓN DE AUTENTICACIÓN DESACTIVADA
+      subtitle: "¡Hola %{handle}!"
     email_reset_update:
-      subject:
-      title:
+      subject: Has solicitado actualizar la dirección de email en %{host}
```
Comment (javier-menendez) [inline config/locales/es.yml:273]:
```suggestion
      subject: Has solicitado actualizar la dirección de correo en %{host}
```

File: config/locales/es.yml [hunk 4]
```diff
@@ -197,131 +197,137 @@ es:
...
+      title: SOLICITUD DE ACTUALIZACIÓN DE EMAIL
```
Comment (javier-menendez) [inline config/locales/es.yml:274]:
```suggestion
      title: SOLICITUD DE ACTUALIZACIÓN DE CORREO
```

File: config/locales/es.yml [hunk 5]
```diff
@@ -197,131 +197,137 @@ es:
...
-      subtitle:
-      body_text:
-      body_html:
-      link_expiration_explanation_html:
+      subject: Por favor confirma la propiedad de la gema %{gem} en %{host}
+      title: CONFIMACIÓN DE PROPIEDAD
+      subtitle: "¡Hola %{handle}!"
+      body_text: Has sido añadido/a como propietario/a de la gema %{gem} por %{authorizer}. Por favor visita el enlace siguiente para confirmarlo.
+      body_html: Has sido añadido/a como propietario/a de la gema <strong><a href="https://rubygems.org/gems/%{gem}">%{gem}</a></strong> por <strong>%{authorizer}</strong>. Por favor visita el enlace siguiente para confirmarlo.
+      link_expiration_explanation_html: Ten en cuenta que este enlace es válido solo durante %{expiry_hours}. Puedes reenviar el email de confirmación desde la página de la gema <strong><a href="https://rubygems.org/gems/%{gem}">%{gem}</a></strong> una vez autenticado.
```
Comment (javier-menendez) [inline config/locales/es.yml:281]:
```suggestion
      link_expiration_explanation_html: Ten en cuenta que este enlace es válido solo durante %{expiry_hours}. Puedes reenviar el correo de confirmación desde la página de la gema <strong><a href="https://rubygems.org/gems/%{gem}">%{gem}</a></strong> una vez autenticado.
```

File: config/locales/es.yml [hunk 6]
```diff
@@ -371,171 +377,171 @@ es:
...
-      Primero tienes que activarla.
-    require_totp_enabled:
-    require_webauthn_enabled:
-    setup_required_html:
-    setup_recommended:
-    strong_mfa_level_required_html:
-    strong_mfa_level_recommended:
-    setup_webauthn_html:
+    session_expired: Ha expirado tu sesión en la página de acceso.
+    require_totp_disabled: La autenticación de múltiples factores basada en OTP ya está activa. Para reconfigurarla debés primero eliminarla.
```
Comment (javier-menendez) [inline config/locales/es.yml:392]:
```suggestion
    require_totp_disabled: La autenticación de múltiples factores basada en OTP ya está activa. Para reconfigurarla debes primero eliminarla.
```

File: config/locales/es.yml [hunk 7]
```diff
@@ -371,171 +377,171 @@ es:
...
+    require_mfa_enabled: No se ha activado la autenticación de múltiples factores. Primero tienes que activarla.
+    require_totp_enabled: No tienes aplicación de autenticación activa. Has de activarla primero.
```
Comment (javier-menendez) [inline config/locales/es.yml:394]:
```suggestion
    require_totp_enabled: No tienes aplicación de autenticación activa. Debes activarla primero.
```

File: config/locales/es.yml [hunk 8]
```diff
@@ -371,171 +377,171 @@ es:
...
+      copied: "[ copiado ]"
       continue: Continuar
       title: Códigos de recuperación
-      copy:
-      saved:
-      note_html:
-      already_generated:
+      copy: "[ copiar ]"
+      saved: Declaro haber guardado mis códigos de recuperación.
+      note_html: "Por favor <strong class='recovery__bold'>copia y guarda</strong> estos códigos de recuperación. Puedes usar estos códigos para accedes y restablecer tu autenticación de múltiples factores si pierdes tu dispositivo. Cada código se puede usar una sola vez."
```
Comment (javier-menendez) [inline config/locales/es.yml:424]:
```suggestion
      note_html: "Por favor <strong class='recovery__bold'>copia y guarda</strong> estos códigos de recuperación. Puedes usar estos códigos para acceder y restablecer tu autenticación de múltiples factores si pierdes tu dispositivo. Cada código se puede usar una sola vez."
```

File: config/locales/es.yml [hunk 9]
```diff
@@ -371,171 +377,171 @@ es:
...
+      otp_or_recovery: OTP o código de recuperación
+      recovery_code: Código de recuperación
+      recovery_code_html: 'Puedes utilizar un <a href="https://guides.rubygems.org/setting-up-multifactor-authentication/#using-recovery-codes-and-re-setup-a-previously-enabled-mfa", class="t-link">código de recuperación</a> válido si has perdido el acceso a tu dispositivo de seguridad o de autenticación de múltiples factores.'
+      security_device: Dispositivo de seguridad
+      verify_code: Verificar código
   notifiers:
     update:
-      success: Has actualizado exitosamente la configuración de tus notificaciones
-        por email.
+      success: Has actualizado exitosamente la configuración de tus notificaciones por email.
```
Comment (javier-menendez) [inline config/locales/es.yml:442]:
```suggestion
      success: Has actualizado exitosamente la configuración de tus notificaciones por correo.
```

File: config/locales/es.yml [hunk 10]
```diff
@@ -371,171 +377,171 @@ es:
...
-      pending:
-      confirm_remove:
+      add_owner: AÑADIR PROPIETARIO
+      name: "PROPIETARIO/A"
+      mfa: ESTADO DE AMF
+      status: ESTADO
+      confirmed_at: CONFIRMADO
+      added_by: AÑADIDO POR
+      action: ACCIÓN
+      email_field: Email / Usuario
```
Comment (javier-menendez) [inline config/locales/es.yml:483]:
```suggestion
      email_field: Correo / Usuario
```

File: config/locales/es.yml [hunk 11]
```diff
@@ -371,171 +377,171 @@ es:
...
+      info: añadir o eliminar propietarios
+      confirmed: Confirmado
+      pending: Pendiente
+      confirm_remove: ¿Seguro que quieres eliminar a este usuario de los propietarios?
     resend_confirmation:
-      resent_notice:
+      resent_notice: Se ha reenviado un mensaje de confirmación a tu correo electrónico
     create:
-      success_notice:
+      success_notice: "Se ha añadido a %{handle} como propietario sin confirmar. Su acceso como propietario se activara cuando haga click en el mensaje de confirmación que se le ha enviado a su correo"
```
Comment (javier-menendez) [inline config/locales/es.yml:492]:
```suggestion
      success_notice: "Se ha añadido a %{handle} como propietario sin confirmar. Su acceso como propietario se activará cuando haga click en el mensaje de confirmación que se le ha enviado a su correo"
```

File: config/locales/es.yml [hunk 12]
```diff
@@ -371,171 +377,171 @@ es:
...
   settings:
     edit:
       title: Editar configuración
-      webauthn_credentials:
-      no_webauthn_credentials:
-      webauthn_credential_note:
-      otp_code:
+      webauthn_credentials: Dispositivo de seguridad
+      no_webauthn_credentials: No tienes dispositivos de seguridad
+      webauthn_credential_note: Un dispositivo de seguridad puede ser cualquier dispositivoque cumpla el estándar FIDO2 como las llaves biométrica y de seguridad.
```
Comment (javier-menendez) [inline config/locales/es.yml:502]:
```suggestion
      webauthn_credential_note: Un dispositivo de seguridad puede ser cualquier dispositivo que cumpla el estándar FIDO2 como las llaves biométrica y de seguridad.
```

File: config/locales/es.yml [hunk 13]
```diff
@@ -371,171 +377,171 @@ es:
...
-      change_avatar:
-      email_awaiting_confirmation: Por favor confirma tu nueva dirección de correo
-        %{unconfirmed_email}
+      change_avatar: Cambiar avatar
+      email_awaiting_confirmation: Por favor confirma tu nueva dirección de correo %{unconfirmed_email}
       enter_password: Por favor introduce tu contraseña
       optional_full_name: Opcional. Será mostrado en tu perfil público
-      optional_twitter_username: Usuario de Twitter opcional. Será mostrado en tu
-        perfil público
+      optional_twitter_username: Usuario de Twitter opcional. Será mostrado en tu perfil público
```
Comment (javier-menendez) [inline config/locales/es.yml:544]:
```suggestion
      optional_twitter_username: Usuario de X opcional. Será mostrado en tu perfil público
```

File: config/locales/es.yml [hunk 14]
```diff
@@ -1,6 +1,6 @@
 ---
 es:
-  credentials_required:
+  credentials_required: Faltan credenciales
```
Comment (acrogenesis) [inline config/locales/es.yml:3]:
```suggestion
  credentials_required: Credenciales requeridas
```
No faltan, se requiren.

File: config/locales/es.yml [hunk 15]
```diff
@@ -24,21 +24,21 @@ es:
   locale_name: Español
   none: Ninguno
   not_found: No encontrado
-  api_key_forbidden:
+  api_key_forbidden: La clave API no tiene permitido el acceso
```
Comment (acrogenesis) [inline config/locales/es.yml:27]:
```suggestion
  api_key_forbidden: La clave API no tiene acceso
```

File: config/locales/es.yml [hunk 16]
```diff
@@ -63,77 +63,77 @@ es:
...
-      mfa:
+      api_keys: Claves de API
+      name: Nombre
+      scopes: Alcances
+      gem: Gema
+      age: Edad
+      last_access: Último accesso
+      action: Acción
+      delete: Borrar
+      confirm: Se va a invalidar la clave de API. ¿Quieres confirmar?
```
Comment (acrogenesis) [inline config/locales/es.yml:104]:
```suggestion
      confirm: Se va a invalidar la clave de API. ¿Estás seguro?
```
In the english version it asks are you sure? Not if you want to confirm
`The API key will become invalid. Are you sure?`

File: config/locales/es.yml [hunk 17]
```diff
@@ -63,77 +63,77 @@ es:
...
+      confirm_all: Todas las claves de API serán invalidadas. ¿Quieres confirmar?
```
Comment (acrogenesis) [inline config/locales/es.yml:105]:
```suggestion
      confirm_all: Todas las claves de API serán invalidadas. ¿Estás seguro?
```

Comment (acrogenesis) [reply config/locales/es.yml:105]: In the english version it asks are you sure? Not if you want to confirm
`All API keys will become invalid. Are you sure?`

File: config/locales/es.yml [hunk 18]
```diff
@@ -154,7 +154,7 @@ es:
       title: Dashboard
   dependencies:
     show:
-      click_to_expand:
+      click_to_expand: Pincha en el icono de flecha para expandir
```
Comment (acrogenesis) [inline config/locales/es.yml:157]:
```suggestion
      click_to_expand: Haz clic en el icono de flecha para expandir
```
`Clic` is used in other parts of the `es.yml`

File: config/locales/es.yml [hunk 19]
```diff
@@ -197,131 +197,137 @@ es:
...
     owner_added:
-      subject_self:
-      subject_others:
-      title:
-      subtitle:
-      body_self_html:
-      body_others_html:
+      subject_self: Has sido añadido/a como propietario/a de la gema %{gem}
+      subject_others: El usuario %{owner_handle} ha sido añadido/a como propietario/a de la gema %{gem}
+      title: AÑADIDO PROPIETARIO
```
Comment (acrogenesis) [inline config/locales/es.yml:285]:
```suggestion
      title: PROPIETARIO AÑADIDO
```

**Comment by xuanxu** [comment]
@javier-menendez & @acrogenesis: Thanks!

**Comment by deivid-rodriguez** [comment]
Hei, I'm late here but sounds like other have already stepped in. This is great!

**Comment by xuanxu** [comment]
Conflict with master resolved. 
Ready!

**Comment by javier-menendez** [comment]
I'm still not sure about the word: mantenedores. Is not a word that is commonly used. I would use other variants like:

Maintainers wanted: Se solicidta mantenimiento
Maintainer: Responsable de mantenimiento

File: config/locales/es.yml [hunk 20]
```diff
@@ -197,131 +197,137 @@ es:
         tested_by: Probado por
         tracking_by: Seguimiento con
         uptime: Uptime
-        verified_by:
-        secured_by:
-        looking_for_maintainers:
+        verified_by: Verificado por
+        secured_by: Protegido por
+        looking_for_maintainers: "Se buscan mantenedores/as"
```
Comment (javier-menendez) [inline config/locales/es.yml:202]:
```suggestion
        looking_for_maintainers: "Se solicita mantenimiento"
```

File: config/locales/es.yml [hunk 21]
```diff
@@ -371,172 +377,172 @@ es:
...
   profiles:
     adoptions:
-      no_ownership_calls:
-      no_ownership_requests:
-      title:
-      subtitle_html:
+      no_ownership_calls: No has creado llamadas a ser propietario para ninguna de tus gemas
+      no_ownership_requests: No has creado ninguna petición para ser propietario
+      title: Adopción
+      subtitle_html: Pide nuevos mantenedores o solicita propietarios <a class="adoption__blog__link" href="https://blog.rubygems.org/2022/01/19/rubygems-adoptions.html">(leer más)</a>
```
Comment (javier-menendez) [inline config/locales/es.yml:538]:
```suggestion
      subtitle_html: Pide nuevos responsables de mantenimiento o solicita propietarios <a class="adoption__blog__link" href="https://blog.rubygems.org/2022/01/19/rubygems-adoptions.html">(leer más)</a>
```

File: config/locales/es.yml [hunk 22]
```diff
@@ -650,113 +654,113 @@ es:
...
       yanked: borrada
   adoptions:
     index:
-      title:
-      subtitle_owner_html:
-      subtitle_user_html:
-      ownership_calls:
-      no_ownership_calls:
+      title: Adopciones
+      subtitle_owner_html: Solicita nuevos mantenedores para %{gem} <a class="adoption__blog__link" href="https://blog.rubygems.org/2022/01/19/rubygems-adoptions.html">(leer más)</a>
```
Comment (javier-menendez) [inline config/locales/es.yml:709]:
```suggestion
      subtitle_owner_html: Solicita nuevos responsables de mantenimiento para %{gem} <a class="adoption__blog__link" href="https://blog.rubygems.org/2022/01/19/rubygems-adoptions.html">(leer más)</a>
```

File: config/locales/es.yml [hunk 23]
```diff
@@ -650,113 +654,113 @@ es:
...
+      subtitle_user_html: Solicita ser propietario de %{gem} <a class="adoption__blog__link" href="https://blog.rubygems.org/2022/01/19/rubygems-adoptions.html">(leer más)</a>
+      ownership_calls: Solicitud de propietarios
+      no_ownership_calls: No hay convocatorias de propietarios para %{gem}. Los dueños de la gema no están buscando nuevos mantenedores.
```
Comment (javier-menendez) [inline config/locales/es.yml:712]:
```suggestion
      no_ownership_calls: No hay convocatorias de propietarios para %{gem}. Los dueños de la gema no están buscando nuevos responsables de mantenimiento.
```

File: config/locales/es.yml [hunk 24]
```diff
@@ -650,113 +654,113 @@ es:
...
-      subtitle_html:
-    share_requirements:
-    note_for_applicants:
-    created_by:
-    details:
-    apply:
-    close:
-    markup_supported_html:
-    create_call:
+      title: Se buscan mantenedores
```
Comment (javier-menendez) [inline config/locales/es.yml:719]:
```suggestion
      title: Se buscan responsables de mantenimiento
```

File: config/locales/es.yml [hunk 25]
```diff
@@ -650,113 +654,113 @@ es:
...
+      subtitle_html: Gemas que buscan nuevos mantenedores <a class="adoption__blog__link" href="https://blog.rubygems.org/2022/01/19/rubygems-adoptions.html">(leer más)</a>
```
Comment (javier-menendez) [inline config/locales/es.yml:720]:
```suggestion
      subtitle_html: Gemas que buscan nuevos responsables de mantenimiento <a class="adoption__blog__link" href="https://blog.rubygems.org/2022/01/19/rubygems-adoptions.html">(leer más)</a>
```

**Comment by acrogenesis** [comment]
> I'm still not sure about the word: mantenedores. Is not a word that is commonly used. I would use other variants like:
> 
> Maintainers wanted: Se solicita mantenimiento Maintainer: Responsable de mantenimiento

I like `Maintainer: Responsable de mantenimiento` but I think `Maintainers wanted: Se solicita mantenimiento` doesn't send the right message. It says we need maintenance, not maintainers. 
At least that's how I read it in Mexican Spanish.

I do use the word `mantenedores` referring to maintainers of OSS

**Comment by xuanxu** [comment]
- I also use `mantenedores` for maintainers of OSS, but I'm ok with `Responsable de mantenimiento` too.

- About `Maintainers wanted: Se solicita mantenimiento`, I'm with @acrogenesis here, I think it changes the original meaning of looking for people not for maintenance.

I'd like to have input from others about `Mantenedores vs Responsable de mantenimiento` to build a bigger consensus. WDYT @deivid-rodriguez ?

**Comment by xuanxu** [comment]
OK, last suggestions merged. 
Code updated to latest master and OIDC strings added.
This should be ready now. 
cc/ @simi @segiddins

**Review by xuanxu — COMMENTED** [review]
OK, last suggestions merged. 
Code updated to latest master and OIDC strings added.
This should be ready now. 
cc/ @simi @segiddins

**Comment by deivid-rodriguez** [comment]
I don't have a strong opinion on Mantenedores vs Responsable de mantenimiento, either works for me!

**Comment by simi** [comment]
Thanks a lot everyone involved. Merging for now, feel free to open additional PR with additional tweaks if needed.

**Pull request merged**